### DISCUSSION TITLE
v0.2.3 - Реліз на Linux + Виправлення

### DISCUSSION THREAD

**Discussion by DanielleTlumach** [yanchukcha/deltarune-ua#17] [General]
[Інструкцію зі встановлення](https://github.com/yanchukcha/deltarune-ua#%D0%B2%D1%81%D1%82%D0%B0%D0%BD%D0%BE%D0%B2%D0%BB%D0%B5%D0%BD%D0%BD%D1%8F-%D0%BC%D0%BE%D0%B4%D0%B0) можна переглянути на головній сторінці.

**УВАГА: Мод працює лише на останній версії гри! (Наразі це 1.10).**

## Що нового:
- Додано порт локалізації на Linux та винятки в коді для його належної роботи.
- У грі відновлено японську локалізацію, бо раніше через українізацію вона була недоступною (переважно це зроблено з поваги до перекладачів, що працювали з Тобі Фоксом, адже ми часто піддивлялися в японську версію за натхненням).

## Що змінилося:
- Виправлено орфографічні помилки.
- Виправлено розмір деяких текстових хмаринок у битві із Королем Хаосу (1 розділ).
- Виправлено розміщення серця в меню вибору варіантів діалогу.
- У хард-моді скорочено кат-сцени перед битвами з босами.
- У хард-моді молоко замінено на йогурт 🙃

<br><b>Дрібні зміни в коді (нічого з цього не впливає на гру):</b>
- Перероблено систему техсимволів для перенесення тексту.

## Відомі проблеми:
- На Android музика може іноді зникати без причини. Досі не знаємо як це повністю полагодити.
- У випадку, якщо порт не працює на вашому телефоні (краші, лаґи тощо), спробуйте натомість встановити «Shaderless»-версію.

**Примітка:** перший та другий розділ закінчено, але досі радимо лишати відгуки щодо перекладу!
Внести в них зміни потребує мінімальних зусиль, тому не соромтеся.

<hr /><em>This discussion was created from the release <a href='https://github.com/yanchukcha/deltarune-ua/releases/tag/v0.2.3'>v0.2.3 - Реліз на Linux + Виправлення</a>.</em>

**Comment by simpleprogamer** [comment]
А що за хард-мод Дельтаруну який ви зазначаєте в змінах цього оновлення? Як я бачив хард-мод є як мод-доповнення, але переклад є на оригінальну версію гри яка є не модифікованою.

**Reply by DanielleTlumach** [reply]
Це якраз і є те мод-доповнення (Йдеться про "Deltarune but Kris is Bleeding"), але в локалізації воно активується лише кодовим словом при створенні персонажа, та й саме доповнення тут вдосконалене: відмінність у різних діалогах з босами, механіка моду повністю працює у 2 розділі, повторні кат-сцени перед битвами із босами скорочені тощо.
